src/data/finsupp/basic.lean

```diff
@@ -1849,6 +1849,10 @@ lemma comap_domain_add :
     comap_domain f v₁ (hf.inj_on _) + comap_domain f v₂ (hf.inj_on _) :=
 by { ext, simp only [comap_domain_apply, coe_add, pi.add_apply] }
 
+lemma comap_domain_smul (n : ℕ) :
+  comap_domain f (n • v₁) (hf.inj_on _) = n • comap_domain f v₁ (hf.inj_on _) :=
```

Comment:
This looks good, but you should generalize from `ℕ` to some `monoid R` with `distrib_mul_action R M`

Reply:
Hmm, Lean does not seem to find a `has_scalar` on `α →₀ M` with the `distrib_action` on `M`.

Reply:
Found it later in the same file: lemma added.